code-challenges.test.js

```diff
@@ -10,60 +12,115 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multipled by 3.
-
+// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
 // a) Create a test with expect statements for each of the variables provided.
-
 const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
 const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
-
-
 // b) Create the function that makes the test pass.
 
+// INSTRUCTOR EXAMPLE completed during class May 20th 
 
 
-// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
 
-// a) Create a test with expect statements for each of the variables provided.
+// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisible by three or not.
 
-const num1 = 15
+// a) Create a test with expect statements for each of the variables provided.
+    // the test should take the const numX and check if it is divisible by 3
+
+describe ("div3", ()=> {
+    it("returns if the number is evenly divisible by 3 or not", () => {
+        const num1 = 15
+    expect(div3(num1)).toEqual("15 is divisible by 3")
+    })
+})
 // Expected output: "15 is divisible by three"
-const num2 = 0
+
+describe ("div3", ()=> {
+    it("returns if the number is evenly divisible by 3 or not", () => {
+        const num2 = 0
+        expect(div3(num2)).toEqual("0 is divisible by 3")
+    })
+})
 // Expected output: "0 is divisible by three"
-const num3 = -7
+
+describe ("div3", ()=> {
+    it("returns if the number is evenly divisible by 3 or not", () => {
+        const num3 = -7
+        expect(div3(num3)).toEqual("-7 is not divisible by 3")
+    })
+})
```

Comment:
Next week let's make this one describe and one it statement with multiple expect statements.